hot-desk/src/main/java/com/exadel/sandbox/config/ModelMapperConfig.java

```diff
@@ -120,6 +128,9 @@ public Equipment convert(MappingContext<EquipmentUpdateDto, Equipment> context)
 
             @Override
             public Seat convert(MappingContext<SeatCreateDto, Seat> context) {
+                if (context.getSource().getFloorId() == null)
+                    throw new CannotBeNullException("Floor id can not be null");
```

Comment:
Let's add javax.validation in our project, instead of this checks

Reply:
Okay I'll google it

Reply:
I don't know about this. But promise I'll add